persistence/sql/migrations/sql/20241023142500000002_drop_unused_indices_sessions.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE INDEX sessions_nid_expires_at_authenticated_at_idx ON sessions (nid ASC, expires_at DESC, created_at DESC); -- Used for listing sessions
```

Comment:
This is a bit problematic because we now have to order by `expires_at, created_at` to avoid reading all rows and use the index. This is a potential breaking change as we previously only ordered by created_at and authenticated_at.

This still needs to be reflected in the persister.

Alternatively, we could consider to add a second index just for `created_at` sorting.

Reply:
Why is `expires_at` in this index?

Reply:
It's used in list sessions and listmysessions when active is set:

```
			if *active {
				q.Where("active = ? AND expires_at >= ?", *active, time.Now().UTC())
			} else {
				q.Where("(active = ? OR expires_at < ?)", *active, time.Now().UTC())
			}
```

So we have `nid, active, expires_at` as keys and I think we need all 3 parameters to find the smallest subset of rows.